pages/bugs.jsx

```diff
@@ -42,27 +46,95 @@ export default function BugsPage() {
     setShowDonated(!showDonated);
   }
 
+  const sortName = () => {
+    setCrittersToShow(
+      [...crittersToShow].sort((critterA, critterB) => {
+        const nameA = critterA.name.toUpperCase();
+        const nameB = critterB.name.toUpperCase();
+        if (isAsc) {
+          if (nameA < nameB) {
+            return -1;
+          }
+          if (nameA > nameB) {
+            return 1;
+          }
+        } else {
+          if (nameA > nameB) {
+            return -1;
+          }
+          if (nameA < nameB) {
+            return 1;
+          }
+        }
+      })
+    );
+    setIsAsc(!isAsc);
+  };
+
+  const sortPrice = () => {
+    setCrittersToShow(
+      [...crittersToShow].sort((critterA, critterB) => {
+        const priceA = parseInt(critterA.price);
+        const priceB = parseInt(critterB.price);
+        if (isAsc) {
+          if (priceA < priceB) {
+            return -1;
+          }
+          if (priceA > priceB) {
+            return 1;
+          }
+        } else {
+          if (priceA > priceB) {
+            return -1;
+          }
+          if (priceA < priceB) {
+            return 1;
+          }
+        }
+      })
+    );
+    setIsAsc(!isAsc);
+  };
+  const setAvailability = e => {
```

Comment:
```suggestion
  const setAvailability = () => {
```

e is not used in the function. Also keep in mind to name this variable, if you were to use it, better. So call it for example event instead of e.